### PR TITLE
Centered checkmarks inside radio-buttons and checkboxes. Improved visibility of Header links in hover state.

### PR DESCRIPTION
**Pull request by zubairatif**
Added the required attribute to email and password in the index.html file.
Centered the circle (dot) inside the checked radio button.
Changed the check mark inside the check box to HTML's heavy check mark sign (#10004) and centered it.
The header links in the light mode during the hover state were not visible due to the brightness filter applied more than 100%  which makes them closer to white and thus lowers the contrast, in the dark theme this was working because of a dark background. But in the light mode, the brightness filter should be applied negatively i.e. lower than 100% to actually make it consistent with the dark theme. 
I did this using a custom property that changes according to the user's theme.

### CODE REVIEW

**Comment by lkhrs** [comment]
Hi, thank you for taking the time to work on this 🙂

Please revert the formatting changes on `index.html`, it makes it difficult to tell what was changed.

> Added the required attribute to email and password in the index.html file.

I don't think this is needed since this is a test page, not an actual form. Is there a reason or use case I'm missing?

> Centered the circle (dot) inside the checked radio button.
> Changed the check mark inside the check box to HTML's heavy check mark sign (#10004) and centered it.

They aren't centered for me in WebKit, Blink, and Gecko. But I think our current checkmark design looks better than the HTML check mark sign. Is there a benefit to using the HTML check mark sign?

Screen Shot 2022-09-18 at 11 58 59

> The header links in the light mode during the hover state were not visible due to the brightness filter applied more than 100% which makes them closer to white and thus lowers the contrast, in the dark theme this was working because of a dark background. But in the light mode, the brightness filter should be applied negatively i.e. lower than 100% to actually make it consistent with the dark theme.
> I did this using a custom property that changes according to the user's theme.

I see what you mean, here's the current version:
Screen Shot 2022-09-18 at 12 11 46
vs. your change:
Screen Shot 2022-09-18 at 12 12 32

Thanks for fixing this issue, I'll cherry pick this commit and merge it.

Let me know about the `required` attribute and checkmark changes.

File: simple.css [hunk 1]
```diff
@@ -53,7 +57,7 @@ progress {
   appearance: none;
   -webkit-appearance: none;
   -moz-appearance: none;
-}
+}
```
Comment (waldyrious) [inline simple.css:60]: There's a stray whitespace change here, should be reverted :)

File: simple.css [hunk 2]
```diff
@@ -448,32 +452,28 @@ input[type="radio"]:checked {
   background-color: var(--accent);
 }
 
+
```
Comment (waldyrious) [inline simple.css:455]: This extra blank like appears to have been added my mistake.

File: simple.css [hunk 3]
```diff
@@ -17,6 +17,8 @@
   --preformatted: #444;
   --marked: #ffdd33;
   --disabled: #efefef;
+  /* Brightness filter is applied on header links in hover state*/
```
Comment (waldyrious) [inline simple.css:20]:
```suggestion
  /* Brightness filter is applied on header links in hover state */
```

File: simple.css [hunk 4]
```diff
@@ -32,6 +34,8 @@
     --code: #f06292;
     --preformatted: #ccc;
     --disabled: #111;
+    /* Brightness filter in hover state is updated to improve visibility in dark-mode*/
```
Comment (waldyrious) [inline simple.css:37]:
```suggestion
    /* Brightness filter in hover state is updated to improve visibility in dark-mode */
```

File: simple.css (hunk 1 above)
Comment (lkhrs) [reply simple.css:60]: I don't see this in upstream after cherry-picking the commits, resolving.

File: simple.css (hunk 2 above)
Comment (lkhrs) [reply simple.css:455]: I don't see this in upstream after cherry-picking the commits, resolving.

**Comment by lkhrs** [comment]
I cherry-picked the commit related to the nav link border and the brightness variable, and I'm moving forward with the existing checkbox and radio styling. Thank you @zubairatif for your work on this, and thank you @waldyrious for reviewing.

**Pull request closed without merging**